irr/include/ISkinnedMesh.h

```diff
@@ -199,6 +199,9 @@ class ISkinnedMesh : public IAnimatedMesh
 	//! Adds a new meshbuffer to the mesh, access it as last one
 	virtual SSkinMeshBuffer *addMeshBuffer() = 0;
 
+	//! Adds a new meshbuffer to the mesh, access it as last one
+	virtual void addMeshBuffer(SSkinMeshBuffer &&meshbuf) = 0;
```

Comment:
this seems unnecessary:
```
foo->addMeshBuffer();
foo->getMeshBuffers().getLast() = std::move(buf);
```

also `SSkinMeshBuffer` is `IReferenceCounted` so using it without a pointer (stack-allocated) can get unsafe and is best avoided.

Reply:
I'll convert to a variant which takes a heap-allocated pointer then, analogeous to `addJoint`.